### PR TITLE
Tracker UI Improvements

### PR DESCRIPTION
**Pull request by elfenheart**
- [x] Global Menu - bigger font
- [x] border right on brand -> 4px
- [x] Cask Tracker search -> don’t allow *
- [x] tracker footer -> add top border
- [x] Change Jump Button Color -> keep it green
- [x] Sort By -> pull right
- [x] Search bar on detail -> make it the same size as main page
- [x] Time Picker -> connect dropdown to button
- [x] lineage no data message -> 150px & same font
- [x] Green arrow lineage -> show all the time
- [x] Runs picker on program popup -> center
- [x] Change date format on integrations
- [x] Integrations: Status & Disabled/Enabled  bold
- [x] Integrations: Padding should be aligned
    - [x] X axis on graph should be at the bottom
    - [x] Confirmation on disable -> bigger box
        - [x] Ok / Cancel
- [x] Remove help page

### CODE REVIEW

**Comment by elfenheart** [comment]
Build - https://builds.cask.co/browse/CDAP-DRC3489-2

File: cdap-ui/app/features/tracker/tracker.less [hunk 1]
```diff
@@ -142,15 +148,13 @@ body.state-tracker {
...
-      padding: 0 20px;
-    }
+    .tracker-filters { padding: 0 20px; }
 
     .tracker-filter-sidebar {
       width: 300px;
       position: absolute;
       background-color: @filter-sidebar-background-color;
-      top: 101px;
+      top: 111px;
```
Comment (ajainarayanan) [inline cdap-ui/app/features/tracker/tracker.less]: Use the variables.

File: cdap-ui/app/features/tracker/tracker.less [hunk 2]
```diff
@@ -200,7 +204,7 @@ body.state-tracker {
 
     .tracker-search-results {
       position: absolute;
-      top: 101px;
+      top: 111px;
```
Comment (ajainarayanan) [inline cdap-ui/app/features/tracker/tracker.less]: Use the variables.

**Comment by ajainarayanan** [comment]
Minor comments :+1: LGTM

**Pull request merged**